### PR TITLE
Update es web translation for 3.9.0

### LINKED ISSUE #775: Update es web translation for 3.9.0
Hi @ignotus666,

We are getting ready for the 3.9.0 release and have created the [next-release](https://github.com/jamulussoftware/jamuluswebsite/tree/next-release) branch ([full diff](https://github.com/jamulussoftware/jamuluswebsite/compare/release..next-release)).

We would be happy if you updated the translations for **es** by **2022-07-22**.

Please

- Start your work in your fork on a branch based on jamuluswebsite's `next-release` branch.
- Update the language-specific files using your favorite editor (or directly on Github),
- New/changed images are listed at the end of this issue. [Generate new URLs](https://github.com/jamulussoftware/jamuluswebsite/tree/release#adding-screenshots) for your image `.inc` files.
- Commit and push your changes to your fork,
- Open a Pull Request with your translations to the **next-release** branch with the subject `Update es web translation for 3.9.0`,
- Link your PR(s) to this issue by including `Fixes #775` in the PR content.

Suggested commit and PR message:
```
Update es web translation for 3.9.0

Fixes #775
```

Feel free to use this Issue to discuss anything you need prior to making any PR (and in your native language if needed).

Further documentation can be found in [TRANSLATING.md](https://github.com/jamulussoftware/jamuluswebsite/blob/next-release/README.md#translating).

Thanks for contributing to Jamulus!

---

**New/Changed screenshots**:

Please [replace with new URLs](https://github.com/jamulussoftware/jamuluswebsite/tree/release#adding-screenshots) for your screenshots here, and use those URLs in your `.inc` files.

settings-profile

server-window-setup

server-window-options

### PR DESCRIPTION
**Pull request by ignotus666**
**Short description of changes**
Update es web translation for 3.9.0

**Context: Fixes an issue? Related issues**
Fixes #775

Comments in previous PR have been addressed.

**Status of this Pull Request**
Should be ready

**What is missing until this pull request can be merged?**

**Does this need translation?**

No

## Checklist

- [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [x] I'm sure that this Pull Request goes to the correct branch

### CODE REVIEW

File: _translator-files/po/es/Getting-Started.po [hunk 1]
```diff
@@ -180,64 +171,48 @@ msgstr "Rellena al menos “Alias/Nombre” y cierra la ventana."
...
 msgid ""
 "<figure>\n"
 "  <img src=\"{% include img/en-screenshots/connection-setup-window.inc %}\" loading=\"lazy\" alt=\"Screenshot of the connection window\" style=\"border: 5px solid grey;\">\n"
 "  <figcaption>Connect to a Server</figcaption>\n"
 "</figure>\n"
 msgstr ""
 "<figure>\n"
-"  <img src=\"{% include img/es-screenshots/connection-setup-window.inc %}\" loading=\"lazy\" alt=\"Screenshot\" style=\"border: 5px solid grey;\">\n"
-"  <figcaption>Conéctate a un servidor</figcaption>\n"
+"  <img src=\"{% include img/es-screenshots/connection-setup-window.inc %}\" loading=\"lazy\" alt=\"Screenshot of the connection window\" style=\"border: 5px solid grey;\">\n"
```
Comment (pljones) [inline _translator-files/po/es/Getting-Started.po]: Shouldn't the screenshot alt (hover) text get translated?

File: _translator-files/po/es/Include-Client-Commands.po [hunk 2]
```diff
@@ -6,28 +6,27 @@
...
 msgid "`-M` or `--mutestream`  Prevent others on a server from hearing what I play                                                      \n"
-msgstr "`-M` o `--mutestream`  Inicia Jamulus en estado silenciado                                                      \n"
+msgstr "`-M` o `--mutestream`  Evita que otros en el servidor escuchen lo que yo toco                                                      \n"
 
 #. type: Plain text
 #: ../wiki/en/Include-Client-Commands.md:3
-#, fuzzy, no-wrap
-#| msgid "`--mutemyown`  Mute me in my personal mix (\"headless\" mode only)                                                      \n"
+#, no-wrap
 msgid "`--mutemyown` Prevent me from hearing what I play in the server mix (headless only)                                                      \n"
```
Comment (pljones) [inline _translator-files/po/es/Include-Client-Commands.po:28]: @gilgongo It looks like there needs to be some checking for trailing spaces here (and in general).

**Review by pljones — APPROVED** [review]
Thanks!  Just one small comment but I'm going to approve.

File: _translator-files/po/es/Getting-Started.po (hunk 1 above)
Comment (ignotus666) [reply _translator-files/po/es/Getting-Started.po]: For some reason I thought that text wasn't shown. Done.

File: _translator-files/po/es/Getting-Started.po [hunk 3]
```diff
@@ -246,19 +221,12 @@ msgstr "## Tocar por primera vez"
...
-#, fuzzy, no-wrap
-#| msgid ""
-#| "<figure>\n"
-#| "  <img src=\"{% include img/en-screenshots/main-screen-medium.inc %}\" loading=\"lazy\" alt=\"Screenshot\" style=\"border: 5px solid grey;\">\n"
-#| "  <figcaption>The main window when you are connected to a sever</figcaption>\n"
-#| "</figure>\n"
+#, no-wrap
 msgid ""
 "<figure>\n"
 "  <img src=\"{% include img/en-screenshots/main-screen-medium.inc %}\" loading=\"lazy\" alt=\"Screenshot\" style=\"border: 5px solid grey;\">\n"
```
Comment (pljones) [inline _translator-files/po/es/Getting-Started.po:232]: And, speaking of alt text, @gilgongo - we really should have alt text for all images.

**Review by pljones — APPROVED** [review]

**Pull request merged**